File: URI_1048.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner scanner = new Scanner(System.in);
		float number;
		number = scanner.nextFloat();
		
		if ( number<=400.00 ) {
			
			float percent = ( (number*15) / 100  );
			float newSalary = ( percent + number );
			
			System.out.printf("Novo salario: %.2f\n",newSalary);
			System.out.printf("Reajuste ganho: %.2f\n",percent);
			System.out.printf("Em percentual: 15 %%\n");
		}
		else if ( number>=400.01 && number<=800.00 ) {
			float percent = ( (number*12) / 100  );
			float newSalary = ( percent + number );
			
			System.out.printf("Novo salario: %.2f\n",newSalary);
			System.out.printf("Reajuste ganho: %.2f\n",percent);
			System.out.printf("Em percentual: 12 %%\n");
		}
		else if ( number>=800.01 && number<=1200.00 ) {
			float percent = ( (number*10) / 100  );
			float newSalary = ( percent + number );
			
			System.out.printf("Novo salario: %.2f\n",newSalary);
			System.out.printf("Reajuste ganho: %.2f\n",percent);
			System.out.printf("Em percentual: 10 %%\n");
		}
		else if ( number>=1200.01 && number<=2000.00 ) {
			float percent = ( (number*7) / 100  );
			float newSalary = ( percent + number );
			
			System.out.printf("Novo salario: %.2f\n",newSalary);
			System.out.printf("Reajuste ganho: %.2f\n",percent);
			System.out.printf("Em percentual: 7 %%\n");
		}
		else if ( number >= 2000.00 ) {
			float percent = ( (number*4) / 100  );
			float newSalary = ( percent + number );
			
			System.out.printf("Novo salario: %.2f\n",newSalary);
			System.out.printf("Reajuste ganho: %.2f\n",percent);
			System.out.printf("Em percentual: 4 %%\n");
		}
 
    }
 
}

File: URI_1048.c
#include<stdio.h>
int main()
{
 float input;
 scanf("%f", &input);
 if (input <= 400.0)
   printf("Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: 15 %%\n", input * 1.15, input * 0.15);
 else if (input <= 800.0)
   printf("Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: 12 %%\n", input * 1.12, input * 0.12);
 else if (input <= 1200.0)
   printf("Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: 10 %%\n", input * 1.10, input * 0.10);
 else if (input <= 2000.0)
   printf("Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: 7 %%\n", input * 1.07, input * 0.07);
 else
   printf("Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: 4 %%\n", input * 1.04, input * 0.04);
 return 0;
}
